modules/CmpdRegAdmin/src/server/routes/CmpdRegAdminRoutes.coffee

```diff
@@ -215,6 +243,29 @@ exports.updateCmpdRegEntity = (req, resp) ->
 			resp.end "Error trying to update #{entityType}: " + error;
 	)
 
+exports.editCmpdRegEntity = (req, resp) ->
+	request = require 'request'
+	config = require '../conf/compiled/conf.js'
+	entityType = req.params.entityType
+	cmpdRegCall = config.all.client.service.cmpdReg.persistence.fullpath + "#{entityType}/edit/" + req.params.id 
```

Comment:
I'd like to discuss this further since the CmpdRegAdmin routes already is tied to the SaltAPIController so I'm not sure if it's good organization-wise to separate the routes. Just because other CReg entities don't use `edit` now doesn't mean it can't be expanded in the future.